config/routes.rb

```diff
@@ -24,6 +24,7 @@
       post :skip, on: :member
       resources :installments, only: [:index, :show]
       resources :subscription_events, only: :index
+      resources :subscription_orders, only: :index
```

Comment:
Can we use `/orders` as the path, for consistency with the path used for a user's order history in the default Solidus backend?

Reply:
Good point: https://github.com/solidusio-contrib/solidus_subscriptions/pull/192/commits/f8f1408e134acbe3860247658bf1949281a4a645